docs/getting-started.md

```diff
@@ -216,16 +233,17 @@ Troubleshooting tips if your EV3 won't boot
 
 ## Step 5: Set up a network connection
 
-There are lots of choices here. Choose the connection option that best suits you
-and your available devices.
+Click on the button below to visit our networking page. There are lots of
+choices here; choose the connection option that best suits you and your
```

Comment:
> In general I notice the same patterns from people of the same (or similar) native language.

This calls for machine learning project - predict the native language of a person from it's written English... 😉